src/renderers/useRenderer.ts

```diff
@@ -1,3 +1,4 @@
+import { flushSync } from 'react-dom';
```

Comment:
nit: there is another import from `react-dom` below, let's combine those.

Reply:
Done